### PR TITLE
[All Hosts] (ui) Update Fabric references for the new Fluent UI and Fabric Core context

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

File: docs/design/using-motion-office-addins.md [hunk 1]
```diff
@@ -50,12 +50,12 @@ Consider the size of the HTML canvas (task pane, dialog box, or content add-in)
 
 |Do|Don't|
 |:-----|:-----|
-| Use [recommended motion durations](https://developer.microsoft.com/fabric#/styles/web/motion). | Don't use exaggerated animations. Avoid creating experiences that embellish and distract your customers.
+| Use [recommended motion durations](https://developer.microsoft.com/fluentui#/styles/web/motion). | Don't use exaggerated animations. Avoid creating experiences that embellish and distract your customers.
 | Follow [recommended easing curves](/windows/uwp/design/motion/timing-and-easing#easing-in-fluent-motion).  |Don't move elements in a jerky or disjointed manner. Avoid anticipations, bounces, rubberband, or other effects that emulate natural world physics.|
 
 ![GIF that shows tiles loading using a gentle fade in against a GIF that shows tiles loading with bounce](../images/add-in-motion-character.gif)
```
Comment (lindalu-MSFT) [inline docs/design/using-motion-office-addins.md:56]: I checked the alt-text in this file and think we should change them all. Suggest replace "against" with "compared to".

File: docs/design/using-office-ui-fabric-react.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 localization_priority: Normal
 ---
 
-# Use Office UI Fabric React in Office Add-ins
+# Use Fluent UI React in Office Add-ins
 
-Office UI Fabric is a JavaScript front-end framework for building user experiences for Office. If you build your add-in using React, consider using Fabric React to create your user experience. Fabric provides several React-based UX components, like buttons or checkboxes, that you can use in your add-in.
-
-This article describes how to create an add-in that's built with React and uses Fabric React components.
+Fluent UI React is a JavaScript front-end framework for building user experiences for Office based on the React framework. It is designed to build experiences that fit seamlessly into a broad range of Microsoft products. It provides robust, up-to-date, accessible React-based components which are highly customizable using CSS-in-JS.
```
Comment (lindalu-MSFT) [inline docs/design/using-office-ui-fabric-react.md:10]:
```suggestion
Fluent UI React is the official open-source JavaScript front-end framework designed to build experiences that fit seamlessly into a broad range of Microsoft products, including Office. It provides robust, up-to-date, accessible React-based components which are highly customizable using CSS-in-JS.
```

**Review by lindalu-MSFT — APPROVED** [review]

File: docs/tutorials/outlook-tutorial.md [hunk 3]
```diff
@@ -311,7 +311,7 @@ Let's start by creating the UI for the dialog itself. Within the **./src** folde
   <!-- Office JavaScript API -->
   <script type="text/javascript" src="https://appsforoffice.microsoft.com/lib/1.1/hosted/office.js"></script>
 
-  <!-- For more information on Office UI Fabric, visit https://developer.microsoft.com/fabric. -->
+  !-- For more information on Fluent UI, visit https://developer.microsoft.com/fluentui. -->
```
[the remark this answers is not in the document]
Comment (Rick-Kirkham) [reply docs/tutorials/outlook-tutorial.md:314]: > Should we tell folks about Fabric Core instead?

No need. They'll see Fabric Core described at that URL. It is one of the two flavors of Fluent UI.

File: docs/tutorials/outlook-tutorial.md [hunk 4]
```diff
@@ -972,7 +972,7 @@ In the project that you've created, the task pane HTML is specified in the file
     <!-- Office JavaScript API -->
     <script type="text/javascript" src="https://appsforoffice.microsoft.com/lib/1.1/hosted/office.js"></script>
 
-    <!-- For more information on Office UI Fabric, visit https://developer.microsoft.com/fabric. -->
+    !-- For more information on Fluent UI, visit https://developer.microsoft.com/fluentui. -->
```
[the remark this answers is not in the document]
Comment (Rick-Kirkham) [reply docs/tutorials/outlook-tutorial.md:975]: > Should we tell folks about Fabric Core instead?

They'll see Fabric Core described at that URL. It is one of the two flavors of Fluent UI.

File: docs/design/fabric-core.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+    For more detailed instructions, see [Fluent UI Icons](https://developer.microsoft.com/fluentui#/styles/web/icons). To find more icons that are available in Fabric Core, use the search feature on the that page. When you find an icon to use in your add-in, be sure to prefix the icon name with `ms-Icon--`.
+
+    For information about font sizes and colors that are available in Fabric Core, see [Typography](https://developer.microsoft.com/fluentui#/styles/web/typography) and the **Colors** table of contents at [Colors](https://developer.microsoft.com/fluentui#/styles/web/colors).
+
+See [Samples](#samples) below.
+
+## Use Office UI Fabric JS components
+
+Add-ins with non-React UIs can also use any of the 31 components from [Office UI Fabric JS](https://github.com/OfficeDev/office-ui-fabric-js), including buttons, dialogs, pickers, and more. See the readme of the repo for instructions.
```
[the remark this answers is not in the document]
Comment (Rick-Kirkham) [reply docs/design/fabric-core.md]: done

File: docs/design/using-motion-office-addins.md (hunk 1 above)
Comment (Rick-Kirkham) [reply docs/design/using-motion-office-addins.md]: Done. I used "beside" instead of "compared to"

File: docs/design/task-pane-add-ins.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Task panes in Office Add-ins
 description: 'Task panes give users access to interface controls that run code to modify documents or emails, or display data from a data source.'
-ms.date: 07/07/2020
+ms.date: 05/12/2020
```
Comment (ElizabethSamuel-MSFT) [inline docs/design/task-pane-add-ins.md:4]:
```suggestion
ms.date: 05/12/2021
```

**Pull request merged**